lib/services/elasticsearch.js

```diff
@@ -1244,16 +1415,29 @@ class ElasticSearch extends Service {
             // document._id should always be equal to mgetResult.docs[idx]._id
             // if the document is not found in ES, then we reject it
             if (mgetResult.docs[idx].found) {
-              esRequest.body.push({index: {_index: index, _type: type, _id: document._id}});
+              esRequest.body.push(
+                {index:
+                  {_index: index,
+                    _type: type,
+                    _id: document._id}
+                });
               esRequest.body.push(document._source);
               toImport.push(document);
             } else {
-              extracted.rejected.push({document, reason: 'cannot replace a non-existing document (use mCreateOrReplace if you need to create non-existing documents)'});
+              extracted.rejected.push(
+                {document,
+                  reason: 'cannot replace a non-existing document ' +
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this concatenation to the use of a template. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3512)